Core/Object Arts/Dolphin/Base/PackageTest.cls

```diff
@@ -363,7 +363,25 @@ testPaxTimestampUpdate
 	self deny: b isRenegade!
 
 testTimestampInitialized
-	self assert: Package new timestamp asInteger = 0! !
+	self assert: Package new timestamp asInteger = 0!
```

Comment:
Rather than do this, let's just delete these two tests. I don't think they are worth the cost of running them.